definitions/npm/react-native-vision-camera_v2.x.x/flow_v0.142.x-/react-native-vision-camera_v2.x.x.js

```diff
@@ -0,0 +1,1358 @@
+
+    /**
+     * Represents the orientation of all Camera Outputs (Photo, Video, and Frame Processor). If this value is not set, the device orientation is used.
+     */
+    orientation?: "portrait" | "portraitUpsideDown" | "landscapeLeft" | "landscapeRight",
+
+    /**
+     * Called when any kind of runtime error occured.
+     */
+    onError?: (error: CameraRuntimeError) => void,
+
+    /**
+     * Called when the camera was successfully initialized.
+     */
+    onInitialized?: () => void,
+
+    /**
+     * Called when a new performance suggestion for a Frame Processor is available - either if your Frame Processor is running too fast and frames are being dropped, or because it is able to run faster. Optionally, you can adjust your `frameProcessorFps` accordingly.
+     */
+    onFrameProcessorPerformanceSuggestionAvailable?: (suggestion: FrameProcessorPerformanceSuggestion) => void,
+
+    /**
+     * A worklet which will be called for every frame the Camera "sees". Throttle the Frame Processor's frame rate with {@linkcode frameProcessorFps}.
+     *
+     * > See [the Frame Processors documentation](https://mrousavy.github.io/react-native-vision-camera/docs/guides/frame-processors) for more information
+     *
+     * Note: If you want to use `video` and `frameProcessor` simultaneously, make sure [`supportsParallelVideoProcessing`](https://mrousavy.github.io/react-native-vision-camera/docs/guides/devices#the-supportsparallelvideoprocessing-prop) is `true`.
+     *
+     * @example ```tsx
+     * const frameProcessor = useFrameProcessor((frame) => {
+     *   'worklet'
+     *   const qrCodes = scanQRCodes(frame)
+     *   console.log(`Detected QR Codes: ${qrCodes}`)
+     * }, [])
+     *
+     * return <Camera {...cameraProps} frameProcessor={frameProcessor} />
+     * ```
+     */
+    frameProcessor?: (frame: Frame) => void,
+
+    /**
+     * Specifies the maximum frame rate the frame processor can use, independent of the Camera's frame rate (`fps` property).
+     *
+     * * A value of `'auto'` (default) indicates that the frame processor should execute as fast as it can, without dropping frames. This is achieved by collecting historical data for previous frame processor calls and adjusting frame rate accordingly.
+     * * A value of `1` indicates that the frame processor gets executed once per second, perfect for code scanning.
+     * * A value of `10` indicates that the frame processor gets executed 10 times per second, perfect for more realtime use-cases.
+     * * A value of `25` indicates that the frame processor gets executed 25 times per second, perfect for high-speed realtime use-cases.
+     * * ...and so on
+     *
+     * If you're using higher values, always check your Xcode/Android Studio Logs to make sure your frame processors are executing fast enough
+     * without blocking the video recording queue.
+     * @default 'auto'
+     */
+    frameProcessorFps?: number | "auto",
+    ...
+  } & ViewProps;
```

Comment:
generally prefer this as it's usually what people intend when they port from TS definitions
```
{
 ...ViewProps,
 // other stuff
}
```

Reply:
Thanks. Fixed in cff3d5b5